src/routes/dashboard/(admin)/drafts/+page.server.ts

```diff
@@ -1,3 +1,8 @@
+import type {
+  DispatchLotteryInterventionArgs,
+  DispatchRoundStartArgs,
+  DispatchUserNotificationArgs,
+} from '$lib/server/email/dispatch.js';
```

Comment:
We use extension-less imports 👌 

```suggestion
} from '$lib/server/email/dispatch';
```